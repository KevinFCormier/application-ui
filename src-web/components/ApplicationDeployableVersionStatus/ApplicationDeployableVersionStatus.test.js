/*******************************************************************************
 * Licensed Materials - Property of IBM
 * (c) Copyright IBM Corporation 2016, 2019. All Rights Reserved.
 *
 * Note to U.S. Government Users Restricted Rights:
 * Use, duplication or disclosure restricted by GSA ADP Schedule
 * Contract with IBM Corp.
 *******************************************************************************/

const React = require('../../../node_modules/react')
const renderer = require('../../../node_modules/react-test-renderer')
const ApplicationDeployableVersionStatus = require('../ApplicationDeployableVersionStatus')
  .default

describe('ApplicationDeployableVersionStatus', () => {
  it('ApplicationDeployableVersionStatus renders correctly.', () => {
    const tree = renderer
      .create(<ApplicationDeployableVersionStatus />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
