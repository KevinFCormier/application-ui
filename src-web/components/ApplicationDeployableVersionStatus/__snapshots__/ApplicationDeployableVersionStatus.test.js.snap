// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationDeployableVersionStatus ApplicationDeployableVersionStatus renders correctly. 1`] = `
<div
  id="ApplicationDeployableVersionStatus"
>
  <div
    className="deployable-versionStatus-header"
  >
    Deployable versions
  </div>
  <div
    className="deployable-version-status-container"
  >
    <div
      className="deployable-left-column"
    >
      <div
        className="bx--tile deployable-left-column-header"
      />
      <div
        className="bx--tile"
      >
        <div
          className="deployable-version-name"
        >
          
           
          
        </div>
        <div
          className="incidents-count"
        >
          0 
          Incidents
        </div>
      </div>
    </div>
  </div>
</div>
`;
